src/pages/Members/index.tsx

```diff
@@ -179,11 +181,10 @@ export default function Members() {
             formatter(r) {
```

Comment:
for discussion: lets check if we can use static typing for parameters and return types. 
This may be tricky given its react components but static typing will benefit the maintainability of the code base in longer run.

Reply:
i think i have a different view, let's discuss in the meeting.